src/client/components/CostWidget.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import DataCard from "./DataCard";
+import { CardGroup, CardGroupHeader } from "./CardGroup";
+
+function CostWidget({ currentCost, remainingCost }) {
+  let totalCost = currentCost + remainingCost;
```

Comment:
use `const` instead of `let` as the value is read-only anyway.

https://tylermcginnis.com/var-let-const/

Reply:
changes made. replaced let with const